tuichain/api/views/loanrequests.py

```diff
@@ -165,6 +165,26 @@ def get_non_validated_loan_requests(request):
         status=HTTP_200_OK
     )
 
+@api_view(["GET"])
+@permission_classes((IsAdminUser,))
+def get_specific_state_loan_requests(request, status):
```

Comment:
I think every user should be able to do this, with some restrictions:
- pending, cancelled and finished and expired can only be seen by the user that requested the loans
- active should be seen by the users that requested it or invested on it
- funding can be seen by everyone